lms/djangoapps/bulk_email/models.py

```diff
@@ -107,7 +107,7 @@ def get_users(self, course_id, user_id=None):
         staff_qset = CourseStaffRole(course_id).users_with_role()
         instructor_qset = CourseInstructorRole(course_id).users_with_role()
         staff_instructor_qset = (staff_qset | instructor_qset)
-        enrollment_qset = User.objects.filter(
+        enrollment_query = models.Q(
```

Comment:
I personally think `Q` objects make the query easier to understand, so I think it's probably an aesthetic preference at this point :)